miio/waterheater.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from typing import Any, Dict
+
+from .click_common import command, format_output
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WaterHeaterStatus:
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
```

Comment:
Please add an example response showing the data to docstring, here's an example: https://github.com/rytilahti/python-miio/blob/master/miio/heater.py#L56